### PR TITLE
Use vendored copy of distutils.version

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Closes #3979. This fixes the issue the current fix (#3936) had for Ansible 2.9 before 2.9.20, and ansible-base 2.10 before 2.10.8.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
plugins/module_util/version.py

### CODE REVIEW

**Review by briantist — APPROVED** [review]
lgtm, only thing I'll ask if this was tested and definitely solves the issue for too old 2.9/2.10 versions?

**Comment by felixfontein** [comment]
I did test it in community.docker with old Ansible versions; here it was only tested with the latest stable-x branches.

**Comment by felixfontein** [comment]
@briantist thanks for reviewing this!

File: plugins/module_utils/version.py [hunk 1]
```diff
@@ -8,13 +8,10 @@
...
 
-try:
-    from ansible.module_utils.compat.version import LooseVersion
-except ImportError:
-    try:
-        from distutils.version import LooseVersion
-    except ImportError as exc:
-        msg = 'To use this plugin or module with ansible-core < 2.11, you need to use Python < 3.12 with distutils.version present'
-        raise_from(ImportError(msg), exc)
+# Once we drop support for Ansible 2.9 and ansible-base 2.10, we can
```
Comment (felixfontein) [inline plugins/module_utils/version.py:12]: This line should have been
```.diff
# Once we drop support for Ansible 2.9 and ansible-base 2.10, we can
# Once we drop support for Ansible 2.9, ansible-base 2.10, and ansible-core 2.11, we can
```
Fixing in follow-up PR #3993.

**Pull request merged**